pages/index.tsx

```diff
@@ -1,24 +1,25 @@
-import Head from 'next/head'
+// https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md#case-i-use-nextjs-and-im-getting-this-error-inside-of-links
```

Comment:
I really appreciate including this documentation along with the eslint-disable! What's the best way to track moving forward, to make sure that if/when the API is updated, we also update this code?

Reply:
discussed in eng sync 🎉